.config/cargo_spellcheck.dic

```diff
@@ -0,0 +1,75 @@
+80
+
+WebAssembly
+Wasm
+Polkadot
+validator
+LLVM
+JIT
+rustup
+miri
+clippy
+hunspell
+ERC
+fuzzer
+1A
+2A
+1B
+2B
+SHA2
+BLAKE2
+KECCAK
+ABI
+FFI
+UI
+crypto
+Dev
+deploy
+layed
+dispatchable
+backend
+SRML
+Params
+encodable
+decodable
+spreaded
+callee
+prefilled
+|
+defrag
+defragments
+defragmentation
+prev
+growable
+th
+codegen
+endian
+deallocation
++
+deallocate
+bitvector
+bitwise
+dereferencing
+recurse
+layouting
+accessors
+kb
+
+Una/M
+unary
+instant/MRYS
+instantiate/DSXGN
+instantiation
+discriminant/S
+comparator/S
+ceil/JG
+interpret/AGVDS
+interpretable/U
+discriminants
+hasher/S
+hashmap/S
+vector/S
+vec/S
+enum/S
+struct/S
+initializer/MS
```

Comment:
Those seem to be very similar to some other `dic` file .... :sweat_smile: Please remove if they have no occurrences in ink!'s codebase.